pkg/registries/registries.go

```diff
@@ -24,7 +24,7 @@ var userRegistriesFile = filepath.Join(os.Getenv("HOME"), ".config/containers/re
 // FIXME: This should be centralized in a global SystemContext initializer inherited throughout the code,
 // not haphazardly called throughout the way it is being called now.
 func SystemRegistriesConfPath() string {
-	if envOverride := os.Getenv("REGISTRIES_CONFIG_PATH"); len(envOverride) > 0 {
```

Comment:
That's a breaking change. Can we continue supporting `REGISTRIES_CONFIG_PATH` but error out if both are set? I'd sleep better.

Reply:
It was never documented and was buried in the code.

Reply:
Yet it was used in many tests for a very long time.  I would feel very uncomfortable to merge as I have no idea who might already rely on it.

Reply:
Rather than warn if both are set, emit a warning if the old one is set recommending to use new name and saying it'll be deprecated in a future version -- then remove it later™ ?

(That said I'm fine with anything, but I agree people like me could have found about it and started using it already even if it's not documented... Although I do believe you can't complain if undocumented features break, which is why I had asked in the first place :D )